ivy/functional/frontends/torch/blas_and_lapack_ops.py

```diff
@@ -91,7 +91,10 @@ def chain_matmul(*matrices, out=None):
 
 @to_ivy_arrays_and_back
 def cholesky(input, upper=False, *, out=None):
-    return ivy.cholesky(input, upper=upper, out=out)
+    temp = ivy.cholesky(input, upper=upper, out=out)
```

Comment:
For future reference, a better naming scheme here would be something like `out` or `ret`.
```suggestion
    ret = ivy.cholesky(input, upper=upper, out=out)
```